app-rating/build.gradle

```diff
@@ -50,6 +50,8 @@ dependencies {
 
     compile ("com.android.support:appcompat-v7:$androidSupportLibraryVersion")
     compile ("com.android.support:support-annotations:$androidSupportLibraryVersion")
+    compile("com.google.collections:google-collections:$googleCollectionsVersion")
```

Comment:
ok, i removed google-collection dependencies and copied just Preconditions class internally